php-classes/Slate/CBL/Calculators/Growth/MostRecentMinusFirst.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Slate\CBL\Calculators\Growth;
+
+use Slate\CBL\StudentCompetency;
+
+class MostRecentMinusFirst implements IGrowthCalculator
+{
+    public static function calculateGrowth(StudentCompetency $StudentCompetency)
+    {
+        $demonstrationData = $StudentCompetency->getDemonstrationData();
+        $growthData = array_filter(array_map(function($demonstrations) use ($StudentCompetency) {
```

Comment:
@nbey you were missing the `use ($StudentCompetency)` bit here and the growth calcs were silently all very off. I'm going to add some vscode extension recommendations in `develop`, next time you sit down make sure you have them all installed and working, intelephense highlighted the issue for me otherwise I would have missed it too